src/js/cypress/Browser.ts

```diff
@@ -253,28 +253,32 @@ export class Browser extends BaseBrowser {
     retry(
         cb: () => Cypress.Chainable<boolean>,
         become?: boolean,
-        delay?: BrowserTimeout,
-        timeout?: BrowserTimeout,
+        delay?: BrowserTimeout | number,
+        timeout?: BrowserTimeout | number,
     ): void;
 
     retry<T>(
         cb: () => Cypress.Chainable<T>,
         become: T,
-        delay?: BrowserTimeout,
-        timeout?: BrowserTimeout,
+        delay?: BrowserTimeout | number,
+        timeout?: BrowserTimeout | number,
     ): void;
 
     retry<T = boolean>(
         cb: () => Cypress.Chainable<T>,
         become?: T,
-        delay: BrowserTimeout = "zero",
-        timeout: BrowserTimeout = TimeoutType.md,
+        delay: BrowserTimeout | number = "zero",
```

Comment:
Numbers here are required in web-app for steps like 
`I see jobs with the following data in {string} status within {string} seconds:`